src/main/java/daomephsta/unpick/impl/constantresolvers/BytecodeAnalysisConstantResolver.java

```diff
@@ -1,31 +1,52 @@
 package daomephsta.unpick.impl.constantresolvers;
 
-import static java.util.stream.Collectors.toSet;
-
-import java.lang.reflect.Modifier;
-import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
 
+import org.jetbrains.annotations.Nullable;
 import org.objectweb.asm.ClassReader;
 import org.objectweb.asm.ClassVisitor;
 import org.objectweb.asm.FieldVisitor;
+import org.objectweb.asm.MethodVisitor;
 import org.objectweb.asm.Opcodes;
 import org.objectweb.asm.Type;
+import org.objectweb.asm.tree.AbstractInsnNode;
+import org.objectweb.asm.tree.FieldInsnNode;
+import org.objectweb.asm.tree.InsnList;
+import org.objectweb.asm.tree.MethodInsnNode;
+import org.objectweb.asm.tree.MethodNode;
 
-import daomephsta.unpick.api.IClassResolver;
-import daomephsta.unpick.api.constantresolvers.IConstantResolver;
-import daomephsta.unpick.impl.LiteralType;
+import daomephsta.unpick.api.classresolvers.IClassResolver;
+import daomephsta.unpick.api.classresolvers.IConstantResolver;
+import daomephsta.unpick.impl.AbstractInsnNodes;
 
 /**
  * Resolves constants by analysing the bytecode of their owners.
  * @author Daomephsta
  */
 public class BytecodeAnalysisConstantResolver implements IConstantResolver {
-	private static final Set<Type> VALID_CONSTANT_TYPES = Arrays.stream(LiteralType.values()).map(LiteralType::getType).collect(toSet());
+	static final Set<Type> VALID_CONSTANT_TYPES = new HashSet<>();
```

Comment:
Why not use `Set.of()`?

Reply:
Because I wrote this before I migrated to newer Java. I'll do another sweep to check for other stuff like this.